### PR TITLE
Updating the Lgtm tooltip Description

### PR DESCRIPTION
**Pull request by yashchaddha**
PR to resolve the issue tagged in Kubernetes repo

Kindly refer
https://github.com/kubernetes/test-infra/issues/30066

This PR is to update the Tooltip of LGTM label as per the docs https://www.kubernetes.dev/docs/onboarding/04-issues-management/#/5

### CODE REVIEW

**Comment by yashchaddha** [comment]
Hi @MadhavJivrajani  and @Priyankasaggu11929 hope you all are well, kindly review my PR as per your convenience and provide your constructive feedback, if any.
Thankyou

File: label_sync/labels.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ larger set of contributors to apply/remove them.
...
 | <a id="do-not-merge/release-note-label-needed" href="#do-not-merge/release-note-label-needed">`do-not-merge/release-note-label-needed`</a> | Indicates that a PR should not merge because it's missing one of the release note labels. <br><br> This was previously `release-note-label-needed`, | prow |  [releasenote](https://git.k8s.io/test-infra/prow/plugins/releasenote) |
 | <a id="do-not-merge/work-in-progress" href="#do-not-merge/work-in-progress">`do-not-merge/work-in-progress`</a> | Indicates that a PR should not merge because it is a work in progress.| prow |  [wip](https://git.k8s.io/test-infra/prow/plugins/wip) |
-| <a id="lgtm" href="#lgtm">`lgtm`</a> | Indicates that a PR is ready to be merged.| reviewers or members |  [lgtm](https://git.k8s.io/test-infra/prow/plugins/lgtm) |
+| <a id="lgtm" href="#lgtm">`lgtm`</a> | 	"Looks good to me", a PR is ready to merge.| reviewers or members |  [lgtm](https://git.k8s.io/test-infra/prow/plugins/lgtm) |
```
Comment (smg247) [inline label_sync/labels.md:170]:
```suggestion
| <a id="lgtm" href="#lgtm">`lgtm`</a> | "Looks good to me", a PR is ready to merge.| reviewers or members |  [lgtm](https://git.k8s.io/test-infra/prow/plugins/lgtm) |
```

**Review by smg247 — COMMENTED** [review]
/ok-to-test

File: label_sync/labels.md [hunk 2]
```diff
@@ -167,7 +167,7 @@ larger set of contributors to apply/remove them.
...
 | <a id="do-not-merge/release-note-label-needed" href="#do-not-merge/release-note-label-needed">`do-not-merge/release-note-label-needed`</a> | Indicates that a PR should not merge because it's missing one of the release note labels. <br><br> This was previously `release-note-label-needed`, | prow |  [releasenote](https://git.k8s.io/test-infra/prow/plugins/releasenote) |
 | <a id="do-not-merge/work-in-progress" href="#do-not-merge/work-in-progress">`do-not-merge/work-in-progress`</a> | Indicates that a PR should not merge because it is a work in progress.| prow |  [wip](https://git.k8s.io/test-infra/prow/plugins/wip) |
-| <a id="lgtm" href="#lgtm">`lgtm`</a> | Indicates that a PR is ready to be merged.| reviewers or members |  [lgtm](https://git.k8s.io/test-infra/prow/plugins/lgtm) |
+| <a id="lgtm" href="#lgtm">`lgtm`</a> | "Looks good to me", a PR is ready to merge.| reviewers or members |  [lgtm](https://git.k8s.io/test-infra/prow/plugins/lgtm) |
```
Comment (Priyankasaggu11929) [inline label_sync/labels.md:170]: Let's keep it consistent with the other existing label descriptions.

```suggestion
| <a id="lgtm" href="#lgtm">`lgtm`</a> | "Looks good to me", indicates that a PR is ready to be merged.| reviewers or members |  [lgtm](https://git.k8s.io/test-infra/prow/plugins/lgtm) |
```

File: label_sync/labels.yaml [hunk 3]
```diff
@@ -314,7 +314,7 @@ default:
       target: both
       addedBy: humans
     - color: 15dd18
-      description: Indicates that a PR is ready to be merged.
+      description: '"Looks good to me", a PR is ready to merge.'
```
Comment (Priyankasaggu11929) [inline label_sync/labels.yaml:317]: same!

```suggestion
      description: '"Looks good to me", indicates that a PR is ready to be merged.'
```

**Review by Priyankasaggu11929 — COMMENTED** [review]
@yashchaddha, thanks for the PR.

Just left a few suggestions, otherwise looks fine to me.

**Comment by yashchaddha** [comment]
Hi @Priyankasaggu11929 Thankyou for your valuable suggestion, I changed the description as per https://www.kubernetes.dev/docs/onboarding/04-issues-management/#/5

Shall i go ahead and commit this suggestion? Which means we would need to change this here too : https://www.kubernetes.dev/docs/onboarding/04-issues-management/#/5

**Comment by Priyankasaggu11929** [comment]
@yashchaddha, so the request in the issue is to basically clarify what `lgtm`  stands for (which the PR changes explain now).

(More context in the slack thread [here](https://kubernetes.slack.com/archives/C09R23FHP/p1689091041984339))

Feel free to go ahead and commit. The onboarding document doesn't need any changes. :slightly_smiling_face:

**Comment by Priyankasaggu11929** [comment]
Also, could you please squash the commits into one. Thank you!

**Comment by yashchaddha** [comment]
> Also, could you please squash the commits into one. Thank you!

Hi @Priyankasaggu11929 Thankyou for the insights, I have squashed the commits into one

**Comment by Priyankasaggu11929** [comment]
Thanks for the changes.

/lgtm
/approve

**Pull request merged**